components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -0,0 +1,58 @@
+import {
+  Accordion,
+  AccordionItem,
+  Checkbox,
+  CheckboxList
+} from 'components/Form';
+
+const VulnerabilitiesGrid = () => {
+  const groups = [
+    {
+      id: 'group1',
+      name: 'this is a group',
+      assets: [{ text: 'a1' }],
+      vulnerabilities: [{ text: 'v1' }, { text: 'v2' }]
+    },
+    {
+      id: 'group2',
+      name: 'this is another group',
+      assets: [{ text: 'a1' }],
+      vulnerabilities: [{ text: 'v1' }, { text: 'v2' }]
+    }
+  ];
```

Comment:
Would it be worth extracting these into like a json file? (I assume it'll be hardcoded in the future?)

Reply:
done!